src/sentry_plugins/trello/plugin.py

```diff
@@ -0,0 +1,278 @@
+            {
+                "name": "board",
+                "label": "Board",
+                "type": "select",
+                "choices": board_choices,
+                "readonly": False,
+                "required": True,
+            },
+            {
+                "name": "list",
+                "depends": ["board"],
+                "label": "List",
+                "type": "select",
+                "has_autocomplete": False,
+                "required": True,
+            },
+        ]
+
+    def get_link_existing_issue_fields(self, request, group, event, **kwargs):
+        """
+        Return the fields needed for linking to an existing issue
+        """
+        return [
+            {
+                "name": "issue_id",
+                "label": "Card",
+                "type": "select",
+                "has_autocomplete": True,
+                "required": True,
+            },
+            {
+                "name": "comment",
+                "label": "Comment",
+                "default": absolute_uri(
+                    group.get_absolute_url(params={"referrer": "trello_plugin"})
+                ),
+                "type": "textarea",
+                "help": ("Leave blank if you don't want to " "add a comment to the Trello card."),
+                "required": False,
+            },
+        ]
+
+    def get_client(self, project):
+        return TrelloApiClient(
+            self.get_option("key", project), token=self.get_option("token", project)
+        )
+
+    def error_message_from_json(self, data):
+        errors = data.get("errors")
+        if errors:
+            return " ".join([e["message"] for e in errors])
+        return "unknown error"
+
+    def create_issue(self, request, group, form_data, **kwargs):
+        client = self.get_client(group.project)
+
+        try:
+            response = client.new_card(
+                idList=form_data["list"], name=form_data["title"], desc=form_data["description"]
+            )
+        except Exception as e:
+            self.raise_error(e)
+
+        return response["shortLink"]
+
+    def link_issue(self, request, group, form_data, **kwargs):
+        client = self.get_client(group.project)
+
+        try:
+            card = client.get_card(form_data["issue_id"])
+        except Exception as e:
+            self.raise_error(e)
+
+        comment = form_data.get("comment")
+        if comment:
+            try:
+                client.create_comment(card["shortLink"], comment)
+            except Exception as e:
+                self.raise_error(e)
+
+        return {"title": card["name"], "id": card["shortLink"]}
+
+    def get_issue_label(self, group, issue, **kwargs):
+        """
+        Return label of the linked issue we show in the UI from the issue string
+        """
+        # the old version of the plugin stores the url in the issue
+        if re.search("\w+/https://trello.com/", issue):
+            short_issue = issue.partition("/")[0]
```

Comment:
Split might be better and more readable here. This is a light suggestion tho.

```suggestion
            short_issue = issue.split("/", 1)[0]
```

Reply:
Also, you can just do `issue =` here and remove a redundant `return` here (with a constant string repeated in two lines).